src/views/WorkspaceDetail.vue

```diff
@@ -139,15 +139,15 @@ export default Vue.extend({
     return {
       localWorkspace: null as string | null,
       editing: false,
-      nodeTables: [] as string[],
-      edgeTables: [] as string[],
-      graphs: [] as string[],
       requestError: null as string | null,
       loading: false,
     };
   },
 
   computed: {
+    nodeTables: () => store.getters.nodeTables,
+    edgeTables: () => store.getters.edgeTables,
+    graphs: () => store.getters.graphs,
```

Comment:
Is it possible to use `mapGetters()`? Or does direct-vuex interfere with that?

Reply:
This is used in place of that, as mapGetters doesn't give any type information, and required workarounds all over the place if used.

Reply:
Gotcha, I thought maybe direct-vuex had some way around that. Oh well.